src/snowflake/cli/plugins/snowpark/zipper.py

```diff
@@ -13,6 +13,7 @@
     "**/.git/*",
     "**/.gitignore",
     "**/.env/*",
+    "**/.snowflake/*",
```

Comment:
`test_zipper` was failing because we have `reset_global_context_and_setup_config_and_logging_levels` which creates log path dir before I can set path in config 🤷

Reply:
let's fix `snowflake_home` fixture as discussed to make this change not necessary